src/librustc_mir/util/pretty.rs

```diff
@@ -600,7 +600,7 @@ pub fn write_allocations<'tcx>(
             Some(GlobalAlloc::Static(did)) if !tcx.is_foreign_item(did) => {
                 match tcx.const_eval_poly(did) {
                     Ok(ConstValue::ByRef { alloc, .. }) => {
-                        write!(w, " ({})", tcx.def_path_debug_str(did))?;
+                        write!(w, " (static: {}, ", tcx.def_path_debug_str(did))?;
```

Comment:
Wait, what's `def_path_debug_str`? Sounds like something used to implement `{:?}`, not the pretty printed path. The name similarly is unfortunate IMO.